<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.26">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Daniel Costas">
<meta name="dcterms.date" content="2025-12-18">

<title>HOLOFISHCOLOUR — Transcriptomic Analysis</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="Transcriptomic_analysis_files/libs/clipboard/clipboard.min.js"></script>
<script src="Transcriptomic_analysis_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="Transcriptomic_analysis_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="Transcriptomic_analysis_files/libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="Transcriptomic_analysis_files/libs/quarto-html/popper.min.js"></script>
<script src="Transcriptomic_analysis_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Transcriptomic_analysis_files/libs/quarto-html/anchor.min.js"></script>
<link href="Transcriptomic_analysis_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Transcriptomic_analysis_files/libs/quarto-html/quarto-syntax-highlighting-587c61ba64f3a5504c4d52d930310e48.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="Transcriptomic_analysis_files/libs/quarto-html/quarto-syntax-highlighting-dark-b758ccaa5987ceb1b75504551e579abf.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<link href="Transcriptomic_analysis_files/libs/quarto-html/quarto-syntax-highlighting-587c61ba64f3a5504c4d52d930310e48.css" rel="stylesheet" class="quarto-color-scheme-extra" id="quarto-text-highlighting-styles">
<script src="Transcriptomic_analysis_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Transcriptomic_analysis_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Transcriptomic_analysis_files/libs/bootstrap/bootstrap-0a61af393cc41664e89a7addda21b354.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="Transcriptomic_analysis_files/libs/bootstrap/bootstrap-dark-1e622d46932456691ade5e7dd01f2178.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="Transcriptomic_analysis_files/libs/bootstrap/bootstrap-0a61af393cc41664e89a7addda21b354.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">


<link rel="stylesheet" href="styles.css">
</head>

<body class="quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const darkModeDefault = authorPrefersDark;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">1</span> Introduction</a></li>
  <li><a href="#objectives" id="toc-objectives" class="nav-link" data-scroll-target="#objectives"><span class="header-section-number">2</span> Objectives</a></li>
  <li><a href="#materials-and-methods" id="toc-materials-and-methods" class="nav-link" data-scroll-target="#materials-and-methods"><span class="header-section-number">3</span> Materials and Methods</a>
  <ul class="collapse">
  <li><a href="#experimental-design-and-sampling" id="toc-experimental-design-and-sampling" class="nav-link" data-scroll-target="#experimental-design-and-sampling"><span class="header-section-number">3.1</span> Experimental design and Sampling</a></li>
  <li><a href="#fishes" id="toc-fishes" class="nav-link" data-scroll-target="#fishes"><span class="header-section-number">3.2</span> Fishes</a>
  <ul class="collapse">
  <li><a href="#sample-types-and-pigmentation-categories" id="toc-sample-types-and-pigmentation-categories" class="nav-link" data-scroll-target="#sample-types-and-pigmentation-categories"><span class="header-section-number">3.2.1</span> Sample Types and Pigmentation Categories</a></li>
  <li><a href="#sample-overview" id="toc-sample-overview" class="nav-link" data-scroll-target="#sample-overview"><span class="header-section-number">3.2.2</span> Sample Overview</a></li>
  </ul></li>
  <li><a href="#sample-collection" id="toc-sample-collection" class="nav-link" data-scroll-target="#sample-collection"><span class="header-section-number">3.3</span> Sample Collection</a>
  <ul class="collapse">
  <li><a href="#skin-sampling" id="toc-skin-sampling" class="nav-link" data-scroll-target="#skin-sampling"><span class="header-section-number">3.3.1</span> Skin Sampling</a></li>
  <li><a href="#intestinal-sampling" id="toc-intestinal-sampling" class="nav-link" data-scroll-target="#intestinal-sampling"><span class="header-section-number">3.3.2</span> 3.2 Intestinal Sampling</a></li>
  </ul></li>
  <li><a href="#rna-extraction-and-quality-assessment" id="toc-rna-extraction-and-quality-assessment" class="nav-link" data-scroll-target="#rna-extraction-and-quality-assessment"><span class="header-section-number">3.4</span> 4. RNA Extraction and Quality Assessment</a>
  <ul class="collapse">
  <li><a href="#rna-isolation" id="toc-rna-isolation" class="nav-link" data-scroll-target="#rna-isolation"><span class="header-section-number">3.4.1</span> 4.1 RNA Isolation</a></li>
  <li><a href="#rna-extraction-quality-metrics" id="toc-rna-extraction-quality-metrics" class="nav-link" data-scroll-target="#rna-extraction-quality-metrics"><span class="header-section-number">3.4.2</span> 4.2 RNA Extraction Quality Metrics</a></li>
  </ul></li>
  <li><a href="#rna-seq-library-preparation-and-sequencing" id="toc-rna-seq-library-preparation-and-sequencing" class="nav-link" data-scroll-target="#rna-seq-library-preparation-and-sequencing"><span class="header-section-number">3.5</span> RNA-Seq Library Preparation and Sequencing</a></li>
  <li><a href="#rna-quality-control-bioanalyzer" id="toc-rna-quality-control-bioanalyzer" class="nav-link" data-scroll-target="#rna-quality-control-bioanalyzer"><span class="header-section-number">3.6</span> 6. RNA Quality Control (Bioanalyzer)</a></li>
  <li><a href="#reference-genome" id="toc-reference-genome" class="nav-link" data-scroll-target="#reference-genome"><span class="header-section-number">3.7</span> 7. Reference Genome</a>
  <ul class="collapse">
  <li><a href="#genome-statistics" id="toc-genome-statistics" class="nav-link" data-scroll-target="#genome-statistics"><span class="header-section-number">3.7.1</span> 7.1 Genome Statistics</a></li>
  </ul></li>
  <li><a href="#bioinformatic-analysis" id="toc-bioinformatic-analysis" class="nav-link" data-scroll-target="#bioinformatic-analysis"><span class="header-section-number">3.8</span> 8. Bioinformatic Analysis</a>
  <ul class="collapse">
  <li><a href="#overview" id="toc-overview" class="nav-link" data-scroll-target="#overview"><span class="header-section-number">3.8.1</span> 8.1 Overview</a></li>
  </ul></li>
  <li><a href="#notes-and-next-steps" id="toc-notes-and-next-steps" class="nav-link" data-scroll-target="#notes-and-next-steps"><span class="header-section-number">3.9</span> 9. Notes and Next Steps</a></li>
  </ul></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="header-section-number">4</span> References</a></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content" id="quarto-document-content">


<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">HOLOFISHCOLOUR — Transcriptomic Analysis</h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Author</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Daniel Costas <a href="mailto:dcostas@iim.csic.es" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0003-4384-8469" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="www.iim.csic.es">
            Instituto de Investigaciones Marinas (IIM-CSIC)
            </a>
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">December 18, 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<section id="introduction" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Introduction</h1>
<p>Pigmentation disorders represent one of the major biological and economic constraints in flatfish aquaculture. In turbot (<em>Scophthalmus maximus</em>), these abnormalities mainly manifest as pseudo-albinism on the ocular side and hypermelanosis on the blind side, leading to a substantial reduction in market value and production efficiency. Such defects originate during early development, particularly around metamorphosis, when asymmetric body patterning and pigmentation are established, and are thought to result from complex interactions between genetic background and environmental factors such as nutrition, light regime, and rearing conditions <span class="citation" data-cites="Matsumoto1990 Hamre2013 Yufera2019">(<a href="#ref-Matsumoto1990" role="doc-biblioref">Matsumoto and Seikai 1990</a>; <a href="#ref-Hamre2013" role="doc-biblioref">Hamre et al. 2013</a>; <a href="#ref-Yufera2019" role="doc-biblioref"><strong>Yufera2019?</strong></a>)</span>.</p>
<p>Despite decades of research, the molecular mechanisms underlying pigmentation disorders in flatfish remain only partially understood. Previous studies have implicated key biological processes including melanocyte differentiation, neural crest cell migration, retinoid and thyroid hormone signaling, and calcium homeostasis as central drivers of pigment cell development and maintenance <span class="citation" data-cites="Seikai1994 Taguchi2008 Yao2022">(<a href="#ref-Seikai1994" role="doc-biblioref"><strong>Seikai1994?</strong></a>; <a href="#ref-Taguchi2008" role="doc-biblioref"><strong>Taguchi2008?</strong></a>; <a href="#ref-Yao2022" role="doc-biblioref"><strong>Yao2022?</strong></a>)</span>. However, much of the available evidence is based on candidate-gene approaches or interspecific comparisons, and comprehensive transcriptome-wide analyses in turbot are still limited.</p>
<p>High-throughput RNA sequencing (RNA-seq) provides a powerful, unbiased approach to investigate complex phenotypes by enabling genome-wide quantification of gene expression. In pigmentation research, transcriptomic studies have successfully identified differentially expressed genes and regulatory pathways associated with melanogenesis, chromatophore development, and pigment cell survival in several flatfish and teleost species <span class="citation" data-cites="Liu2016 Li2021">(<a href="#ref-Liu2016" role="doc-biblioref">Liu, Wang, and Zheng 2016</a>; <a href="#ref-Li2021" role="doc-biblioref"><strong>Li2021?</strong></a>)</span>. These approaches are particularly well suited to disentangle the polygenic architecture of pigmentation disorders, which are unlikely to be explained by single major-effect loci alone.</p>
<p>Within the HOLOFISHCOLOUR project, transcriptomic characterization constitutes a key component for linking genomic variation to functional biological mechanisms. By comparing gene expression profiles between normally pigmented and malpigmented turbot across biologically relevant tissues, this task aims to identify molecular signatures associated with pigmentation anomalies and to complement ongoing genomic, histological, and microbiome analyses. Together, these integrative results will contribute to a holistic, multi-omics understanding of pigmentation disorders in turbot aquaculture.</p>
</section>
<section id="objectives" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Objectives</h1>
<p>El objetivo principal de esta tarea fue realizar un perfil transcriptómico global de la piel de rodaballos (<em>Scophthalmus maximus</em>) con fenotipo pigmentario normal como malpigmentado, para realizar:</p>
<ul>
<li>Caracterización global de los patrones de expresión génica asociados a los fenotipos de pigmentación.</li>
<li>Identificar los genes diferencialmente expresados entre individuos bien pigmentados y mal pigmentados.</li>
<li>Explorar los mecanismos y rutas metabólicas potencialmente involucradas las malformaciones pigmentarias en rodaballo.</li>
</ul>
<ol type="1">
<li><p>RNA isolation and quality</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/Figure_1_skin_sampling.png" class="img-fluid figure-img"></p>
<figcaption>Figure 1. Skin sampling for transcriptomic analysis. A–B: Sampling areas in pseudo-albino and normally pigmented turbot. Areas outlined with yellow dashed lines correspond to non-pigmented skin, whereas areas outlined in red correspond to pigmented skin. C: Skin incision and sample collection.</figcaption>
</figure>
</div></li>
<li><p>RNA library construction and high-throughput sequencing.<br>
</p></li>
<li><p>Bioinformatic processing and differential expression analysis.</p></li>
</ol>
<hr>
</section>
<section id="materials-and-methods" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Materials and Methods</h1>
<section id="experimental-design-and-sampling" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="experimental-design-and-sampling"><span class="header-section-number">3.1</span> Experimental design and Sampling</h2>
</section>
<section id="fishes" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="fishes"><span class="header-section-number">3.2</span> Fishes</h2>
<p>Three individuals per phenotype (normal vs malpigmented) were selected from tanks <strong>E6</strong> and <strong>E8</strong>, where pigmentation differences were most pronounced.</p>
<ul>
<li><strong>Tank E6:</strong> offspring from two different parental pairs.<br>
</li>
<li><strong>Tank E8:</strong> half-sib family (shared mother, two different fathers).</li>
</ul>
<p>From each individual, <strong>skin and intestine</strong> samples were collected.</p>
<hr>
<section id="sample-types-and-pigmentation-categories" class="level3" data-number="3.2.1">
<h3 data-number="3.2.1" class="anchored" data-anchor-id="sample-types-and-pigmentation-categories"><span class="header-section-number">3.2.1</span> Sample Types and Pigmentation Categories</h3>
<ul>
<li><strong>Normally pigmented fish:</strong>
<ul>
<li>Pigmented skin (dark) on the ocular side</li>
<li>Non-pigmented skin (light) on the blind side</li>
</ul></li>
<li><strong>Malpigmented fish:</strong>
<ul>
<li>Pigmented and non-pigmented skin on both ocular and blind sides</li>
</ul></li>
</ul>
<hr>
</section>
<section id="sample-overview" class="level3" data-number="3.2.2">
<h3 data-number="3.2.2" class="anchored" data-anchor-id="sample-overview"><span class="header-section-number">3.2.2</span> Sample Overview</h3>
<p><strong>Table 1. Samples selected for transcriptomic analysis</strong></p>
<table class="caption-top table">
<colgroup>
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Sample ID</th>
<th style="text-align: center;">Tissue</th>
<th style="text-align: center;">Tank</th>
<th style="text-align: center;">Individual No.</th>
<th style="text-align: center;">Pigmentation phenotype</th>
<th style="text-align: center;">Surface</th>
<th style="text-align: center;">Pigmentation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">P31</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">46</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P32</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">46</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P33</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">52</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P34</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">52</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P35</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">70</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P36</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">70</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P41</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">142</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P42</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">142</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P43</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">142</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P44</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E8</td>
<td style="text-align: center;">131</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P45</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">130</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P46</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">130</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P47</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">130</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P48</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">130</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P49</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">26</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P50</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">26</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Ocular</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P51</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">26</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Dark</td>
</tr>
<tr class="even">
<td style="text-align: center;">P52</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">E6</td>
<td style="text-align: center;">26</td>
<td style="text-align: center;">Malpigmented</td>
<td style="text-align: center;">Blind</td>
<td style="text-align: center;">Light</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P1G</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">56</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
</tr>
<tr class="even">
<td style="text-align: center;">P2G</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">34</td>
<td style="text-align: center;">Normal</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
</tr>
<tr class="odd">
<td style="text-align: center;">PNKD</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">Ambicolor</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
</tr>
<tr class="even">
<td style="text-align: center;">PNKV</td>
<td style="text-align: center;">Skin</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">Ambicolor</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
</tr>
<tr class="odd">
<td style="text-align: center;">IT1G</td>
<td style="text-align: center;">Intestine</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
</tr>
<tr class="even">
<td style="text-align: center;">ITPNK</td>
<td style="text-align: center;">Intestine</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
<td style="text-align: center;">—</td>
</tr>
</tbody>
</table>
<hr>
</section>
</section>
<section id="sample-collection" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="sample-collection"><span class="header-section-number">3.3</span> Sample Collection</h2>
<section id="skin-sampling" class="level3" data-number="3.3.1">
<h3 data-number="3.3.1" class="anchored" data-anchor-id="skin-sampling"><span class="header-section-number">3.3.1</span> Skin Sampling</h3>
<p>Fish were maintained at −80 °C and placed on ice prior to sampling to prevent rapid thawing. While still frozen, approximately <strong>20 mg of skin tissue</strong> were excised using sterile dissection tools.</p>
<p>Muscle tissue was carefully removed by gentle scraping to avoid contamination.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/Figure_1_skin_sampling.png" class="img-fluid figure-img"></p>
<figcaption><strong>Figure 1.</strong> Skin sampling for transcriptomic analysis. A–B: Sampling areas in pseudo-albino and normally pigmented turbot. Areas outlined with yellow dashed lines correspond to non-pigmented skin, whereas areas outlined in red correspond to pigmented skin. C: Skin incision and sample collection.</figcaption>
</figure>
</div>
<hr>
</section>
<section id="intestinal-sampling" class="level3" data-number="3.3.2">
<h3 data-number="3.3.2" class="anchored" data-anchor-id="intestinal-sampling"><span class="header-section-number">3.3.2</span> 3.2 Intestinal Sampling</h3>
<p>Intestinal samples were obtained by ventral dissection. The visceral mass was removed, and intestinal contents were gently expelled before collecting the posterior intestine segment.</p>
<p><em>Figure 2. Intestinal sampling procedure.</em></p>
<hr>
</section>
</section>
<section id="rna-extraction-and-quality-assessment" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="rna-extraction-and-quality-assessment"><span class="header-section-number">3.4</span> 4. RNA Extraction and Quality Assessment</h2>
<section id="rna-isolation" class="level3" data-number="3.4.1">
<h3 data-number="3.4.1" class="anchored" data-anchor-id="rna-isolation"><span class="header-section-number">3.4.1</span> 4.1 RNA Isolation</h3>
<p>Total RNA was extracted using a <strong>TRIzol™ + Phasemaker™ Tubes</strong> protocol, followed by purification with the <strong>RNeasy Mini Kit (Qiagen)</strong>.</p>
<p>RNA concentration was measured using <strong>NanoDrop™ One</strong>, and integrity was assessed by <strong>1.5% agarose gel electrophoresis</strong>.</p>
<table class="caption-top table">
<thead>
<tr class="header">
<th style="text-align: center;">Sample</th>
<th style="text-align: center;">DNA concentration (ng/µl)</th>
<th style="text-align: center;">A260/A280</th>
<th style="text-align: center;">A260/A230</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">P31</td>
<td style="text-align: center;">1142.7</td>
<td style="text-align: center;">2.14</td>
<td style="text-align: center;">2.10</td>
</tr>
<tr class="even">
<td style="text-align: center;">P32</td>
<td style="text-align: center;">813.4</td>
<td style="text-align: center;">2.14</td>
<td style="text-align: center;">2.18</td>
</tr>
<tr class="odd">
<td style="text-align: center;">IT11</td>
<td style="text-align: center;">534.3</td>
<td style="text-align: center;">2.13</td>
<td style="text-align: center;">2.00</td>
</tr>
<tr class="even">
<td style="text-align: center;">P33</td>
<td style="text-align: center;">338.9</td>
<td style="text-align: center;">2.12</td>
<td style="text-align: center;">2.24</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P34</td>
<td style="text-align: center;">440.5</td>
<td style="text-align: center;">2.10</td>
<td style="text-align: center;">2.27</td>
</tr>
<tr class="even">
<td style="text-align: center;">IT12</td>
<td style="text-align: center;">974.0</td>
<td style="text-align: center;">2.13</td>
<td style="text-align: center;">2.36</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P35</td>
<td style="text-align: center;">299.3</td>
<td style="text-align: center;">2.12</td>
<td style="text-align: center;">1.08</td>
</tr>
<tr class="even">
<td style="text-align: center;">P36</td>
<td style="text-align: center;">308.8</td>
<td style="text-align: center;">2.13</td>
<td style="text-align: center;">2.18</td>
</tr>
<tr class="odd">
<td style="text-align: center;">IT13</td>
<td style="text-align: center;">367.6</td>
<td style="text-align: center;">2.13</td>
<td style="text-align: center;">1.99</td>
</tr>
<tr class="even">
<td style="text-align: center;">P37</td>
<td style="text-align: center;">191.5</td>
<td style="text-align: center;">2.10</td>
<td style="text-align: center;">2.20</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P38</td>
<td style="text-align: center;">209.5</td>
<td style="text-align: center;">2.14</td>
<td style="text-align: center;">2.05</td>
</tr>
<tr class="even">
<td style="text-align: center;">P39</td>
<td style="text-align: center;">827.3</td>
<td style="text-align: center;">2.15</td>
<td style="text-align: center;">2.31</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P40</td>
<td style="text-align: center;">359.5</td>
<td style="text-align: center;">2.14</td>
<td style="text-align: center;">2.17</td>
</tr>
<tr class="even">
<td style="text-align: center;">IT14</td>
<td style="text-align: center;">822.9</td>
<td style="text-align: center;">2.14</td>
<td style="text-align: center;">2.31</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P41</td>
<td style="text-align: center;">583.6</td>
<td style="text-align: center;">2.10</td>
<td style="text-align: center;">2.29</td>
</tr>
<tr class="even">
<td style="text-align: center;">P42</td>
<td style="text-align: center;">506.5</td>
<td style="text-align: center;">2.10</td>
<td style="text-align: center;">2.15</td>
</tr>
<tr class="odd">
<td style="text-align: center;">P43</td>
<td style="text-align: center;">588.0</td>
<td style="text-align: center;">2.10</td>
<td style="text-align: center;">2.33</td>
</tr>
<tr class="even">
<td style="text-align: center;">P44</td>
<td style="text-align: center;">1028.3</td>
<td style="text-align: center;">2.13</td>
<td style="text-align: center;">2.14</td>
</tr>
<tr class="odd">
<td style="text-align: center;">IT15</td>
<td style="text-align: center;">727.2</td>
<td style="text-align: center;">2.10</td>
<td style="text-align: center;">2.29</td>
</tr>
</tbody>
</table>
<hr>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/gel_bands.png" class="img-fluid figure-img"></p>
<figcaption>Figure X. Integrity of total RNA assessed by agarose gel electrophoresis (1.5%, 100 mV, 30 min). Each gel corresponds to a batch of samples listed in the table above.</figcaption>
</figure>
</div>
</section>
<section id="rna-extraction-quality-metrics" class="level3" data-number="3.4.2">
<h3 data-number="3.4.2" class="anchored" data-anchor-id="rna-extraction-quality-metrics"><span class="header-section-number">3.4.2</span> 4.2 RNA Extraction Quality Metrics</h3>
<p><strong>Table 2. RNA concentration and purity metrics</strong></p>
<table class="caption-top table">
<thead>
<tr class="header">
<th>Sample</th>
<th>Concentration (ng/µl)</th>
<th>A260/A280</th>
<th>A260/A230</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>P31</td>
<td>1142.7</td>
<td>2.14</td>
<td>2.10</td>
</tr>
<tr class="even">
<td>P32</td>
<td>813.4</td>
<td>2.14</td>
<td>2.18</td>
</tr>
<tr class="odd">
<td>P33</td>
<td>338.9</td>
<td>2.12</td>
<td>2.24</td>
</tr>
<tr class="even">
<td>P34</td>
<td>440.5</td>
<td>2.10</td>
<td>2.27</td>
</tr>
<tr class="odd">
<td>P35</td>
<td>299.3</td>
<td>2.12</td>
<td>1.08</td>
</tr>
<tr class="even">
<td>P36</td>
<td>308.8</td>
<td>2.13</td>
<td>2.18</td>
</tr>
</tbody>
</table>
<hr>
</section>
</section>
<section id="rna-seq-library-preparation-and-sequencing" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="rna-seq-library-preparation-and-sequencing"><span class="header-section-number">3.5</span> RNA-Seq Library Preparation and Sequencing</h2>
<p>Transcriptome sequencing (RNA-Seq) was performed through the <em>Plant and Animal Whole Genome Sequencing (WOBI)</em> service (<a href="#tbl-rnaseq-specs" class="quarto-xref">Table&nbsp;1</a>), outsourced to Novogene (quotation <strong>NVUK2024021911</strong>). Messenger RNA (mRNA) libraries were constructed using a strand-specific protocol, preserving transcript orientation information and enabling accurate downstream expression analyses.</p>
<p>Library preparation was based on poly(A) enrichment to selectively capture mature mRNA transcripts. Sequencing was carried out on an Illumina NovaSeq X Plus platform using a paired-end strategy with a read length of 150 base pairs (2 × 150 bp, PE150).</p>
<p>Sequencing quality criteria were defined to ensure high data reliability, with at least 85% of bases achieving a Phred quality score ≥ Q30. A minimum sequencing output of 6 Gb per sample was guaranteed, providing sufficient depth and coverage for subsequent differential gene expression analyses. In total, 50 RNA-Seq libraries were successfully sequenced.</p>
<div id="tbl-rnaseq-specs" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-rnaseq-specs-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: RNA-Seq specifications
</figcaption>
<div aria-describedby="tbl-rnaseq-specs-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<thead>
<tr class="header">
<th style="text-align: left;">Parameter</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Sequencing platform</td>
<td style="text-align: left;">Illumina NovaSeq X Plus</td>
</tr>
<tr class="even">
<td style="text-align: left;">Read configuration</td>
<td style="text-align: left;">Paired-end (2 × 150 bp)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Library preparation</td>
<td style="text-align: left;">Strand-specific mRNA (poly(A) enrichment)</td>
</tr>
<tr class="even">
<td style="text-align: left;">Quality threshold</td>
<td style="text-align: left;">≥ 85% of bases with Q score ≥ 30</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Data output</td>
<td style="text-align: left;">≥ 6 Gb per sample</td>
</tr>
<tr class="even">
<td style="text-align: left;">Number of libraries</td>
<td style="text-align: left;">50</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<hr>
</section>
<section id="rna-quality-control-bioanalyzer" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="rna-quality-control-bioanalyzer"><span class="header-section-number">3.6</span> 6. RNA Quality Control (Bioanalyzer)</h2>
<p>RNA integrity was evaluated using the <strong>Agilent 5400 Bioanalyzer</strong>, generating RNA Integrity Number (RIN) values.</p>
<ul>
<li>Most samples showed <strong>RIN values between 7.0 and 9.8</strong>, suitable for RNA-seq.</li>
<li>Sample <strong>IT12 (RIN = 3.8)</strong> failed QC and was excluded.</li>
<li>Sample <strong>IT16 (RIN = 6.0)</strong> showed moderate degradation and was also discarded.</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/Figure_20_RNA_QC.png" class="img-fluid figure-img"></p>
<figcaption>RNA Bioanalyzer profiles</figcaption>
</figure>
</div>
<p><em>Figure 3. Representative Bioanalyzer electropherograms.</em></p>
<hr>
</section>
<section id="reference-genome" class="level2" data-number="3.7">
<h2 data-number="3.7" class="anchored" data-anchor-id="reference-genome"><span class="header-section-number">3.7</span> 7. Reference Genome</h2>
<p>RNA-seq analyses were conducted using the <strong>Scophthalmus maximus ASM1334776v1</strong> genome assembly (Ensembl release 113).</p>
<section id="genome-statistics" class="level3" data-number="3.7.1">
<h3 data-number="3.7.1" class="anchored" data-anchor-id="genome-statistics"><span class="header-section-number">3.7.1</span> 7.1 Genome Statistics</h3>
<p><strong>Table 4. Reference genome summary</strong></p>
<table class="caption-top table">
<thead>
<tr class="header">
<th>Feature</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Assembly</td>
<td>ASM1334776v1</td>
</tr>
<tr class="even">
<td>Genome size</td>
<td>556,696,898 bp</td>
</tr>
<tr class="odd">
<td>Coding genes</td>
<td>21,263</td>
</tr>
<tr class="even">
<td>Non-coding genes</td>
<td>12,989</td>
</tr>
<tr class="odd">
<td>Gene transcripts</td>
<td>76,329</td>
</tr>
<tr class="even">
<td>Annotation source</td>
<td>Ensembl</td>
</tr>
</tbody>
</table>
<p>The following files were used:</p>
<ul>
<li><strong>Genome FASTA:</strong> <code>Scophthalmus_maximus.ASM1334776v1.dna.toplevel.fa</code></li>
<li><strong>Annotation GTF:</strong> <code>Scophthalmus_maximus.ASM1334776v1.113.gtf</code></li>
</ul>
<hr>
</section>
</section>
<section id="bioinformatic-analysis" class="level2" data-number="3.8">
<h2 data-number="3.8" class="anchored" data-anchor-id="bioinformatic-analysis"><span class="header-section-number">3.8</span> 8. Bioinformatic Analysis</h2>
<section id="overview" class="level3" data-number="3.8.1">
<h3 data-number="3.8.1" class="anchored" data-anchor-id="overview"><span class="header-section-number">3.8.1</span> 8.1 Overview</h3>
<p>The bioinformatic workflow included:</p>
<ol type="1">
<li>Raw read quality control<br>
</li>
<li>Alignment to the reference genome<br>
</li>
<li>Gene-level quantification<br>
</li>
<li>Differential expression analysis<br>
</li>
<li>Functional enrichment analysis</li>
</ol>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/Flowchart_RNAseq_pipeline.png" class="img-fluid figure-img"></p>
<figcaption>RNA-seq bioinformatic workflow</figcaption>
</figure>
</div>
<p><em>Figure 4. RNA-seq bioinformatic analysis pipeline.</em></p>
<hr>
</section>
</section>
<section id="notes-and-next-steps" class="level2" data-number="3.9">
<h2 data-number="3.9" class="anchored" data-anchor-id="notes-and-next-steps"><span class="header-section-number">3.9</span> 9. Notes and Next Steps</h2>
<p>This transcriptomic dataset provides a solid foundation for:</p>
<ul>
<li>Integrating GWAS candidate genes</li>
<li>Cross-validating pigmentation pathways</li>
<li>Comparative analyses with other flatfish species</li>
<li>Multi-omics integration within the HOLOFISHCOLOUR framework</li>
</ul>
<p>Further refinement will include: - Differential expression results - GO / KEGG enrichment - Integration with phenotypic pigmentation metrics</p>
<hr>
</section>
</section>
<section id="references" class="level1 unnumbered" data-number="4">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">4 References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-Hamre2013" class="csl-entry" role="listitem">
Hamre, K., M. Yúfera, I. Rønnestad, and C. Boglione. 2013. <span>“Metamorphosis and Malpigmentation in Flatfish.”</span> <em>Aquaculture Nutrition</em> 19: 601–13.
</div>
<div id="ref-Liu2016" class="csl-entry" role="listitem">
Liu, F., X. Wang, and Y. Zheng. 2016. <span>“Transcriptomic Analysis of Pigmentation in Flatfish.”</span> <em>BMC Genomics</em> 17: 100.
</div>
<div id="ref-Matsumoto1990" class="csl-entry" role="listitem">
Matsumoto, J., and T. Seikai. 1990. <span>“Asymmetric Pigmentation and Metamorphosis in Flatfish.”</span> <em>Aquaculture</em> 84: 179–90.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>